server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/WebAdminUtils.java

```diff
@@ -41,6 +41,12 @@
 public class WebAdminUtils {
 
     public static WebAdminServer createWebAdminServer(MetricFactory metricFactory, Routes... routes) {
+        WebAdminServer webAdminServer = createNotStartedWebAdminServer(metricFactory, routes);
```

Comment:
I think that it should be the opposite: by default it should not start and you could add a method to createAndStart.
BTW, if `start` returns `this` having two methods is pointless IMO.